lstchain/reco/r0_to_dl1.py

```diff
@@ -731,7 +731,8 @@ def r0_to_dl1(
                                 lg_peak_sample = np.argmax(stacked_waveforms, axis=-1)[1]
 
                             if good_ring:
-                                fill_muon_event(muon_parameters,
+                                fill_muon_event(None,
```

Comment:
although here we are inside `not is_simu`, why passing a `None` when we could pass the `parameters` data frame?

Reply:
ok, I was too lazy to check that. Will do.

Reply:
> ok, I was too lazy to check that. Will do.

Just added -1. for the mc_energy  for the real data case, to avoid possible issues. I see no advantage in passing a container that is not used anyway.